src/common/catalog-entities/kubernetes-cluster.ts

```diff
@@ -50,7 +50,7 @@ export class KubernetesCluster extends CatalogEntity<CatalogEntityMetadata, Kube
       },
     ];
 
-    if (this.status.active) {
+    if (this.status.phase == "connected") {
```

Comment:
Why this change? 🤔

Reply:
Mixed concepts. Active not equals to connected, it's something shared between any kind of catalog entities. Connect/Disconnect seems a more proper relation to me.